grpc/server/admin_test.go

```diff
@@ -99,3 +166,123 @@ func TestAdminServer_GetAccessToken(t *testing.T) {
 		})
 	}
 }
+
+func TestAdminServer_ListAccessTokens(t *testing.T) {
+	tests := []struct {
+		name              string
+		token             *commonv1.AccessToken
+		errCode           codes.Code
+		errMsg            string
+		mockQuerierReturn func(*commonv1.AccessToken) ([]db.ListAccessTokensRow, error)
```

Comment:
Does this func need an argument? I don't think it's actually ever used.

Reply:
all tests are using a token for setting up the `mockQuerierReturn` data and for the final assertion used at the client result (all admin requests are returning one or more token objects, except for the `revokeToken`).